sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -84,7 +84,6 @@ class Message(object):  # pylint: disable=too-many-public-methods,too-many-insta
     :keyword str to: The `to` address used for auto_forward chaining scenarios.
     :keyword str reply_to: The address of an entity to send replies to.
     :keyword str reply_to_session_id: The session identifier augmenting the `reply_to` address.
-    :keyword str encoding: The encoding for string data. Default is UTF-8.
```

Comment:
I'm inclining towards just stopping to document the `encoding` keyword since we just want to hide it for now - thoughts?

Reply:
good to me

Reply:
Seconded.